packages/start/fs-router/router.js

```diff
@@ -386,17 +386,20 @@ export function stringifyAPIRoutes(
    * @return {string}
    */
   function _stringifyRoutes(/** @type {(RouteConfig)[]} */ routes) {
+    const methodNotFound = jsFile.addNamedImport("methodNotFound", "solid-start/api/utils");
```

Comment:
Not 100% sure about this. Could also just write `./utils` instead of `solid-start/api/utils`.